eth_typing/abi.py

```diff
@@ -20,12 +20,26 @@
 Decodable = Union[bytes, bytearray]
 """Binary data to be decoded."""
 
-ABI_COMPONENT_DEPRECATION_MSG = "`{0}` is deprecated, use `ABIComponent` instead."
+ABI_COMPONENT_DEPRECATION_MSG = f"`{0}` is deprecated, use {1} instead."
```

Comment:
It doesn't look like this is being used anywhere?

Reply:
oops, good catch!